third_party/move/move-compiler-v2/src/pipeline/uninitialized_use_checker.rs

```diff
@@ -0,0 +1,381 @@
+// Copyright © Aptos Foundation
```

Comment:
Is there a reason why this need to be in the same PR? Just asking.

Reply:
No, I originally wanted to do stacked PRs but figured out that this cannot be done with forked repo model.

I am removing this checker from this PR and will create a separate PR for it.